sdk/webpubsub/azure-messaging-webpubsub/src/test/java/com/azure/messaging/webpubsub/WebPubSubServiceAsyncClientTests.java

```diff
@@ -392,8 +391,10 @@ public void testAadCredential() {
     }
 
     @Test
-    @DoNotRecord
+    @Disabled("This test requires a connectionId with SEND_TO_GROUP permission.")
```

Comment:
Can we disable this only in live mode instead of disabling this entirely?

Reply:
It did have DoNotRecord label before.
So, it doesn't run in CI, I think.

Reply:
I can check.